fastchat/serve/gradio_web_server.py

```diff
@@ -534,7 +536,6 @@ def bot_response(
             "finish": round(finish_tstamp, 4),
             "state": state.dict(),
             "ip": get_ip(request),
-            "images": images_hash,
```

Comment:
I moved the logic for the image hashes to [conversation](https://github.com/lm-sys/FastChat/blob/ee9e830b3727d17ae9f9c5dafb7d96a902b90d2d/fastchat/conversation.py#L504) because when we do a vote on the response, we save the entire state but that might not have the image hashes. So, we have to move the image hashing logic to `conversation.py`